package.json

```diff
@@ -32,6 +33,7 @@
     "commitmsg": "commitlint -E GIT_PARAMS"
   },
   "main": "lib/App.js",
+  "module": "es/App.js",
```

Comment:
why this?

Reply:
I mean how does this work

Reply:
related maybe to https://github.com/kiwicom/smart-faq/pull/601/files#diff-e56633f72ecc521128b3db6586074d2cR42

Reply:
that's de-facto standard for shipping code with ES2015 modules, if the consumer understand it, this will be used, otherwise `main`

Reply:
see this proposal: https://github.com/dherman/defense-of-dot-js/blob/master/proposal.md

Reply:
build for client with webpack understands it => will use `module`
e.g. node.js in SSR which  doesn't support ES2015 modules => will use `main`